svgelements/svgelements.py

```diff
@@ -7333,19 +7333,32 @@ def index_to_path_index(self, index):
             return slice(start, stop, step)
         return self._numeric_index(index)
 
-    def bbox(self):
+    def bbox(self, transformed=True, with_stroke=False):
         """returns a bounding box for the input Path"""
+        if transformed:
+            return Path(self).bbox(transformed=transformed, with_stroke=with_stroke)
+
         segments = self._path._segments[self._start : self._end + 1]
         bbs = [seg.bbox() for seg in segments if not isinstance(Close, Move)]
         try:
             xmins, ymins, xmaxs, ymaxs = list(zip(*bbs))
         except ValueError:
             return None  # No bounding box items existed. So no bounding box.
-        xmin = min(xmins)
-        xmax = max(xmaxs)
-        ymin = min(ymins)
-        ymax = max(ymaxs)
-        return xmin, ymin, xmax, ymax
+
+        if with_stroke and self._path.stroke_width is not None:
+            if transformed:
+                delta = float(self._path.implicit_stroke_width) / 2.0
```

Comment:
This code can no longer function. if transformed we returned early. We cannot at this point in the program flow have `transformed==true`.

Reply:
D'oh. I will fix.